fiboa_cli/datasets/nl.py

```diff
@@ -0,0 +1,101 @@
+from ..convert_utils import convert as convert_
+
+URI = "https://service.pdok.nl/rvo/referentiepercelen/atom/downloads/referentiepercelen.gpkg"
+
+ID = "nl"
+TITLE = "Field boundaries for The Netherlands"
+DESCRIPTION = """
+A field block (Dutch: "Referentieperceel"), formerly known as "AAN" (Agrarisch Areaal Nederland),
+is a contiguous agricultural area surrounded by permanent boundaries, which is cultivated by one or
+more farmers with one or more crops, is fully or partially set aside or is fully or partially
+taken out of production.
+
+The following field block types exist:
+
+- Woods (Hout)
+- Agricultural area (Landbouwgrond)
+- Other (Overig)
+- Water (Water)
+
+We filter on "Agricultural area" in this converter.
+For crop data, look at BasisRegistratie gewasPercelen (BRP)
+"""
+BBOX = [2.35417303, 50.71447164, 7.5553525, 55.66948102]
+
+PROVIDER_NAME = "RVO / PDOK"
+PROVIDER_URL = "https://www.pdok.nl/introductie/-/article/referentiepercelen"
+ATTRIBUTION = None
+LICENSE = "CC0"  # http://creativecommons.org/publicdomain/mark/1.0/
+
+ADD_COLUMNS = {
+    "determination_datetime": "2023-06-15T00:00:00Z"
+}
+
+EXTENSIONS = []
+COLUMN_MIGRATIONS = []
```

Comment:
```suggestion
COLUMN_MIGRATIONS = [
    # Projection is in CRS 28992 (RD New), this is the area calculation method of the source organization
    "area": lambda column: column / 10000
]
```

The migration can happen here

Reply:
Thanks for reviewing. I could not get this to work. "area" is not a column of the dataset (so it's not an `item` of gdf). The column is introduced with `gdf['area'] = gdf.area / 10000` by accessing the [geopandas area](https://geopandas.org/en/stable/docs/reference/api/geopandas.GeoSeries.area.html) property. The area is determined by calculating the surface of the associated geometry in EPSG:28992, which is exactly the same method the government agency uses in this domain.

Reply:
Oh, I see. That was sloppy on my side. Thanks for the clarification, then we keep it in the migration, indeed. I'll set all related comments to solved.